### ISSUE TITLE
Invite TSC members in the Google Calendar event for meetings

### ISSUE THREAD

**Issue by mmarchini** [nodejs/TSC#1133]
In the TSC meeting events that are added to the Node.js calendar, it would be nice if members were added as guests, that way the meetings will show in our professional/personal (whichever folks want to use) calendars without us having to duplicate the event ourselves.

There are three ways I could see this being done:

1. Invite each TSC member individually. It's a bit of a hassle, although I think you could just copy/paste from the TSC members list on nodejs/node README
2. Create a TSC Google Group. It's one more thing to onboard/offboard, but with the benefit that inviting the Google Group will invite everyone at once, and the list will keep up to date with onboarding/offoboarding
3. Invite tsc@nodejs.com, although I haven't tested this method and have no idea if it would work

There might also be possible to automate something based on the list in https://github.com/nodejs/email, but I think if we can avoid yet another custom automation that would be better.

Thoughts?

**Comment by mcollina** [comment]
I'm really +1 to this

**Comment by BridgeAR** [comment]
That would definitely be helpful!

+1

**Comment by richardlau** [comment]
+1. I've been duplicating and it's a bit of a hassle keeping up, e.g. when meetings are canceled.

**Comment by cjihrig** [comment]
+1. This would be good.

**Comment by Trott** [comment]
Let's try #3 and see if that works? @mhdawson What do you think? (I think you're the one that sets up those calendar events?)

**Comment by mhdawson** [comment]
@Trott anybody with calendar access can setup/modify the calendar entries.  I'm willing to try 3 but I can't understand how that would work.  I will add tsc@iojs.org as I think that is the mailing list that is active so we can see what happens.

I think 2 is likely the most practical. We create/move/change the invites (there are now 3 or 4) that inviting people individually would be a hassle.

**Comment by mhdawson** [comment]
From what I see so far, we would need to create a calendar for tsc@iojs.org and then people would have to add that  calendar to their set which I don't think solves the issue.

@Trott I see you are already a calendar maintainer and could experiment with option 2). @mmarchini I'm also happy to add you as a calendar maintainer if you want to experiment as well.

**Comment by mhdawson** [comment]
Info on becoming a calendar maintainer if anybody want to be added - https://github.com/nodejs/admin#nodejs-project-calendar

**Comment by mhdawson** [comment]
@BethGriggs mentioned she got the calendar invite ok. Maybe what I see is just because I'm the creator of the invite and it will work for everybody else?  Let me know.  If adding tsc@iojs.org works that would be great.

**Comment by mmarchini** [comment]
I received an invite, not sure if it'll be kept in sync with the our main one though. I'm happy to be added as a calendar maintainer and do some experimentation

**Comment by mhdawson** [comment]
@mmarchini can you PR yourself into the calendar maintainer list as per https://github.com/nodejs/admin#nodejs-project-calendar and then I'll add you

**Comment by mmarchini** [comment]
@mhdawson PR opened to add me as calendar maintainer: https://github.com/nodejs/admin/pull/653

**Comment by mmarchini** [comment]
I see some folks have accepted/declined the invite to the 4am-5am pacific meeting, I think if we send the invite to tsc@ we receive an email, and if we accept/decline the invite on that email it'll go to our calendar, but otherwise it wont. 

I'm going to try creating a Google Group, if folks don't mind I can add everyone to it, otherwise I would like a couple volunteers to ensure that inviting the Google Group to the event will also add it to our calendars.

**Comment by aduh95** [comment]
@mmarchini The invite seems to work, thanks! The calendar event includes a link to a Google Meet meeting, can we change that to link to the Zoom meeting instead?

**Comment by mhdawson** [comment]
The invites worked for me as well.

**Comment by mmarchini** [comment]
Yes, we can change to the Zoom link instead.

**Comment by mcollina** [comment]
I think this is working well right now, removing the tsc-agenda label.

**Comment by mmarchini** [comment]
Made some progress on this today:

1. Changed the Google Meet link to Zoom link. Whenever changing the meeting time we need to remember to do the same.

<img width="419" alt="Screen Shot 2022-03-23 at 1 13 35 PM" src="https://user-images.githubusercontent.com/4048656/159787489-c75c4b04-ede8-4386-a530-e13095d7cc75.png">

2. Added everyone that I could to the Google Group. I was unable to add four folks (Google Groups would either not recognize the email, or would give an error or auto-reject when I tried). Since we're not using the Google Group for anything besides the Calendar invite, it's not critical, but I can reach out to folks whom it didn't work if needed.

I also opeend a PR to include Google Group in our onboarding doc.
